export_layers.py

```diff
@@ -10,77 +10,125 @@
 
 sys.path.append('/usr/share/inkscape/extensions')
 import inkex
+#inkex.localization.localize()
 
 Layer = collections.namedtuple('Layer', ['id', 'label', 'tag'])
 Export = collections.namedtuple('Export', ['visible_layers', 'file_name'])
 
-FIXED = '[fixed]'
+FIXED  = '[fixed]'
+F      = '[f]'
 EXPORT = '[export]'
+E      = '[e]'
 
-PNG = 'png'
+PDF = 'pdf'
 SVG = 'svg'
+PNG = 'png'
 JPEG = 'jpeg'
 
 
 class LayerExport(inkex.Effect):
     def __init__(self):
         inkex.Effect.__init__(self)
-        self.OptionParser.add_option('-o', '--output-dir',
+        self.arg_parser.add_argument('-o', '--output-source',
                                      action='store',
-                                     type='string',
-                                     dest='output_dir',
+                                     type=str,
+                                     dest='output_source',
                                      default='~/',
-                                     help='Path to an output directory')
-        self.OptionParser.add_option('-f', '--file-type',
+                                     help='Path to source file in output directory')
+        self.arg_parser.add_argument('--output-subdir',
+                                     action='store',
+                                     type=str,
+                                     dest='output_subdir',
+                                     default='',
+                                     help='name of sub-directory in output path')
+        self.arg_parser.add_argument('-f', '--file-type',
                                      action='store',
-                                     type='choice',
-                                     choices=(PNG, SVG, JPEG),
+                                     choices=(PDF, PNG, SVG, JPEG),
                                      dest='file_type',
                                      default='png',
                                      help='Exported file type')
-        self.OptionParser.add_option('--fit-contents',
+        self.arg_parser.add_argument('--fit-contents',
                                      action='store',
-                                     type='inkbool',
+                                     type=str,
```

Comment:
`type=str`, but `default=False`. Should it be `type=bool` here?

Reply:
I did this a while ago but I believe there was something going wrong with the bool type. 

If I remember well, the argument that was passed was parsed as  a string which would always

Reply:
A typical opt-in boolean is usually implemented as
```python
parser.add_argument('--enable-something', action='store_true', help='Enable something')
```
Such argument is passed as
```bash
./script.py --enable-something  # The options.enable_something == True
./script.py  # The options.enable_something == False
```
(Similar but with opposite effect, an opt-out boolean has `action='store_false').

I'll revise the parser code later.